archivr.R

```diff
@@ -220,7 +308,7 @@ set_api_key <- function (key) {
 #'
 #' @param url The url to extract urls.
 #' @return a vector of urls.
-get_urls_from_webpage <- function (url) {
+extract_urls_from_webpage <- function (url) {
```

Comment:
this function doesn't extract internal/relative links (because of the `startsWith` command). I think generally thats the right choice. We should
a) either document it (I think this is probably the better option) or
b) add an option to this that it would inherit from `archiv.fromURL`

What do you think?

Reply:
I think documenting would be better for now. If all things work properly, I think fixing the README will be part of the PR that turns this into a package.  (Packaging will require some serious refactoring of the code).